packages/component-library/src/components/plmg-avatar/test/plmg-avatar.spec.tsx

```diff
@@ -49,6 +49,37 @@ describe('plmg-avatar', () => {
       </plmg-avatar>
     `);
   });
+  it('renders without user defined colors when an image is provided', async () => {
```

Comment:
This is just a thought; I haven't tested it. Can we test the presence and value of a css property? 
I wonder if we can get access to our styles using [`window.getComputedStyle(el)`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle)
Could you have a look into it?

Reply:
Yep. That works. I've committed two tests that check the background color is applied conditionally. Tried to write a test to check the color prop on the <plmg-svg-icon> component is working, but I can't select the svg element inside .... yet - further research needed!